app/views/pageflow/config/_editor_seeds.json.jbuilder

```diff
@@ -15,11 +15,14 @@ json.file_types Pageflow.config.file_types do |file_type|
     json.collection_name nested_file_type.collection_name
   end
 end
+
 json.file_importers Pageflow.config.file_importers do |file_importer|
-  json.importer_name file_importer.name
-  json.logo_source file_importer.logo_source
-  json.authentication_provider file_importer.authentication_provider
-  json.authentication_required file_importer.authentication_required current_user
+  if file_importer.feature_enabled?(@entry)
+    json.importer_name file_importer.name
+    json.logo_source file_importer.logo_source
+    json.authentication_provider file_importer.authentication_provider
+    json.authentication_required file_importer.authentication_required current_user
+  end
```

Comment:
File import does not need to know about the feature concept. If you use `Pageflow.config_for(entry)`, the importers registered inside a disabled feature will not be present. All of the methods like `feature_name`, `feature_enabled?`, `find_enabled_by_name` can then be removed.

Reply:
The spec testing that disabled importers are not included is still missing.